test/e2e/installplan_e2e_test.go

```diff
@@ -2401,12 +2535,13 @@ var _ = Describe("Install Plan", func() {
 				require.True(GinkgoT(), ok, "couldn't find %v in fetched CRD versions: %#v", key, fetchedCRDVersions)
 			}
 		})
+
 		It("UpdatePreexistingCRDFailed", func() {
 
```

Comment:
super off topic: we really need to refactor these top-level It specs (that also deserve a more descriptive name).

Reply:
One more ticket for the CI epic =D